README.md

```diff
@@ -48,13 +75,13 @@ After the installation is completed, the following command will download the
 environment variable:
 
 ```shell
-setup-envtest use -p env | source
```

Comment:
Perhaps this is a fish thing? I know picking any one shell syntax is arbitrary, but I'd guess more readers will find use in the bash example than fish, and fish users are probably used to translating from bash 😉

Reply:
Oh yes it is 😄 
Looking at this, `eval (setup-envtest use -p env)` seems to behave the same as `setup-envtest use -p env | source`. TIL